Modules/+Sources/@VelocityLaser/VelocityLaser.m

```diff
@@ -211,6 +229,40 @@ function delete(obj)
                 rethrow(err)
             end
         end
+        
+        function val = set_wheel_ip(obj,val,~)
+            err = obj.connect_driver('wheel', 'ArduinoServo', val, obj.wheel_pin);
+            if isempty(obj.wheel)
+                if ~isempty(err)
+                    rethrow(err)
+                end
+                val = 'No Server';
+                return
+            end
+            if ~isempty(err)
+                rethrow(err)
+            end
+        end
+        function val = set_wheel_pin(obj,val,~)
+            err = obj.connect_driver('wheel', 'ArduinoServo', obj.wheel_ip, val);
+            if ~isempty(err)
+                rethrow(err)
+            end
+        end
+        function val = set_wheel_pos(obj,val,~)
+            if ~isempty(obj.wheel)
+    %             meta = obj.get_meta_pref('wheel_pos');
```

Comment:
what's this? the third argument to the set method is the meta pref

Reply:
Oh!